pkg/scheduler/internal/queue/scheduling_queue_test.go

```diff
@@ -58,26 +58,31 @@ var (
 	lowPriority, midPriority, highPriority = int32(0), int32(100), int32(1000)
 	mediumPriority                         = (lowPriority + highPriority) / 2
 
-	highPriorityPodInfo = framework.NewPodInfo(
+	highPriorityPodInfo = newPodInfo(
 		st.MakePod().Name("hpp").Namespace("ns1").UID("hppns1").Priority(highPriority).Obj(),
 	)
-	highPriNominatedPodInfo = framework.NewPodInfo(
+	highPriNominatedPodInfo = newPodInfo(
 		st.MakePod().Name("hpp").Namespace("ns1").UID("hppns1").Priority(highPriority).NominatedNodeName("node1").Obj(),
 	)
-	medPriorityPodInfo = framework.NewPodInfo(
+	medPriorityPodInfo = newPodInfo(
 		st.MakePod().Name("mpp").Namespace("ns2").UID("mppns2").Annotation("annot2", "val2").Priority(mediumPriority).NominatedNodeName("node1").Obj(),
 	)
-	unschedulablePodInfo = framework.NewPodInfo(
+	unschedulablePodInfo = newPodInfo(
 		st.MakePod().Name("up").Namespace("ns1").UID("upns1").Annotation("annot2", "val2").Priority(lowPriority).NominatedNodeName("node1").Condition(v1.PodScheduled, v1.ConditionFalse, v1.PodReasonUnschedulable).Obj(),
 	)
-	nonExistentPodInfo = framework.NewPodInfo(
+	nonExistentPodInfo = newPodInfo(
 		st.MakePod().Name("ne").Namespace("ns1").UID("nens1").Obj(),
 	)
-	scheduledPodInfo = framework.NewPodInfo(
+	scheduledPodInfo = newPodInfo(
 		st.MakePod().Name("sp").Namespace("ns1").UID("spns1").Node("foo").Obj(),
 	)
 )
 
+func newPodInfo(pod *v1.Pod) *framework.PodInfo {
```

Comment:
```suggestion
func mustNewPodInfo(pod *v1.Pod) *framework.PodInfo {
```

and Fatal if there is an error